app/templates/public.hbs

```diff
@@ -38,11 +38,17 @@
       <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="three wide column public">
```

Comment:
@prateekj117 Please see if your issue is solved by replacing this line with :

```hbs
<div class="{{#if device.isMobile 'three wide column public' ''}}
```

Reply:
@prateekj117 make this change.